labs/CiscoContainerPlatform-101/2.md

```diff
@@ -13,6 +13,39 @@ Notice in the Cluster details page, a VIP has already been assigned to the Maste
 Each Cisco Container platform installation comes with integrated Open API documentation, powered by OpenAPI (Swagger tooling).
 You can access this [HERE (https://10.10.20.110/2/swaggerapi/)](https://10.10.20.110/2/swaggerapi/), where you see all the currently supported platform APIs.
 
+Note: There is a known issue that the Swagger API page may fail to load.
+
+And here is workaround for the issue if you are using Chrome browser:
+
+
+1. Open the swagger UI using the following URL format:```https://<ccp_ui_ip>/2/swaggerapi```
+    
+2. Click the **three dots** icon at the upper right corner of the menu bar, and then choose **More Tools > Developer Tools**.
+
+    The Developer Tools panel appears.
+
+3. Click **Sources > Overrides > Select folder for overrides**.
+
+4. Select a newly created empty folder to store the local overrides.
+
+5. In the notification that appears just below the URL bar, click **Allow** to give Developer Tools full access to the selected folder.
+
+6. Check the **Enable Local Overrides** checkbox.
+
+7. Click the **Page** tab and click on the index file.
+
+    The index file appears in the right pane.
+ ![](assets/images/swagger_not_loading.png)
+8. Edit the index file to add the following line after the first line:
+```
+<meta http-equiv="Content-Security-Policy" content="default-src *; style-src 'self' 'unsafe-inline'; script-src * 'unsafe-inline' 'unsafe-eval'"> 
+```
+
+9. Save the file using CTRL+S and ensure Swagger Page is fully loaded.
```

Comment:
"Save the file and ensure that the Swagger page loads correctly."

Reply:
Updated 2.md